### PR TITLE
[package_info]Update README.md to reflect iOS issue.

### PR DESCRIPTION
**Pull request by pambrose**
## Description

Changes to the version string in 'pubspec.yaml' are not reflected in the package_info value on iOS. 

As noted [here](https://github.com/flutter/flutter/issues/20761), package_info on iOS requires the Xcode build folder to be rebuilt after changes to the version string in `pubspec.yaml`.

This PR does not fix the problem, but warns users about the need to rebuild the Xcode build folder. 

## Related Issues

No related issues.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by cyanglaz** [comment]
Thank you for creating this PR! Please update the description and related issue in this PR.

**Comment by pambrose** [comment]
I am sorry, but I am not sure what you are asking me to do. I believe you already prefixed the description with "[package_info]" What more needs to be done? Thanks.

**Comment by cyanglaz** [comment]
@pambrose No worries. So I only prefixed the title of the PR. There is a description and related issue in the PR body that you might want to fill out, so we know what is this PR about.

**Comment by pambrose** [comment]
Sorry, now I understand what you wanted.

File: packages/package_info/README.md [hunk 1]
```diff
@@ -30,6 +30,13 @@ PackageInfo.fromPlatform().then((PackageInfo packageInfo) {
 });
 ```
 
+## IOS-specific Instructions
+
+As noted [here](https://github.com/flutter/flutter/issues/20761), package_info on iOS
```
Comment (cyanglaz) [inline packages/package_info/README.md]: Nits:
```suggestion
As noted [here](https://github.com/flutter/flutter/issues/20761#issuecomment-493434578), package_info on iOS 
```

**Pull request merged**